build.gradle

```diff
@@ -155,12 +153,12 @@ dependencies {
     compile 'net.jcip:jcip-annotations:1.0'
 
     // Force to use the latest org.springframework.security
-    compile group: 'org.springframework.security', name: 'spring-security-core', version: '5.6.6'
-    compile group: 'org.springframework.security', name: 'spring-security-config', version: '5.6.6'
-    compile group: 'org.springframework.security', name: 'spring-security-web', version: '5.6.6'
-    compile group: 'org.springframework.security', name: 'spring-security-crypto', version: '5.6.6'
+    compile group: 'org.springframework.security', name: 'spring-security-core', version:versions.springSecurity
+    compile group: 'org.springframework.security', name: 'spring-security-config', version:versions.springSecurity
+    compile group: 'org.springframework.security', name: 'spring-security-web', version:versions.springSecurity
+    compile group: 'org.springframework.security', name: 'spring-security-crypto', version:versions.springSecurity
 
-    compile 'uk.gov.hmcts.reform:properties-volume-spring-boot-starter:0.0.4'
```

Comment:
changed to use 'com.github.hmcts' to fix local build issue. Relevant ticket: [CCD-2451](https://tools.hmcts.net/jira/browse/CCD-2451)